cmd/dex/serve.go

```diff
@@ -74,7 +74,7 @@ func serve(cmd *cobra.Command, args []string) error {
 		errMsg string
 	}{
 		{c.Issuer == "", "no issuer specified in config file"},
-		{len(c.Connectors) == 0 && !c.EnablePasswordDB, "no connectors supplied in config file"},
+		{len(c.StaticConnectors) == 0 && !c.EnablePasswordDB, "no connectors supplied in config file"},
```

Comment:
Probably should remove this, right? The connectors can be specified in the storage.